.travis.yml

```diff
@@ -3,14 +3,10 @@ language: scala
 scala:
   - 2.10.6
   - 2.11.8
+  - 2.12.0
```

Comment:
is the policy to build+test against the earliest patch-version for each minor-version, to ensure maximum compatibility?

Reply:
I have just copied from 

```yaml
matrix:
  include:
  - scala: 2.12.0
    jdk: oraclejdk8
```